index.html

```diff
@@ -1,14 +1,322 @@
+              ipsum, dolor sit amet consectetur adipisicing elit. Aliquid aut maiores impedit. Quidem inventore
+              neque
+              laborum autem quasi sapiente nostrum totam fugiat ut veniam? Eaque delectus iure temporibus nulla
+              veritatis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Necessitatibus, illo quo nulla
+              ullam
+              et aspernatur tempora eaque, voluptatibus harum magnam voluptatum illum aliquam sunt. Veniam maxime
+              vel
+              dignissimos quibusdam voluptates. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Maiores
+              corporis suscipit rerum iure fugiat doloremque illum quidem consectetur, officiis illo, quis est
+              harum!
+              Nesciunt consequuntur, fuga aut dolor delectus neque? Lorem ipsum dolor sit amet consectetur
+              adipisicing
+              elit. Laudantium, illum ipsum repellat commodi quae beatae libero itaque. Quaerat aut a magnam
+              dignissimos dolore iste, at dicta in, aliquid consequatur deserunt!</p>
+            <a href='./continue-reading.html'><button class='tell-me-button'>Continue Reading</button></a>
+          </div>
+        </div>
+      </div>
+
+
+
+
+      <!-- Featured Content SideBar -->
+      <section class="sidebar">
+        <div class="dates">
+          <div class='date-post'>
+          </div>
+          <a href='./blog-post.html'><button class='tell-me-button'>JULY 30</button></a>
+          <h3 class='date-title'>JUST ANOTHER <span id='blue-color'>POST</span></h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates culpa pariatur eos, tempora
+            nihil
+            maiores, ullam omnis impedit libero eveniet deleniti atque error?</p>
+        </div>
+        <div class="dates">
+          <div class='date-post'>
+          </div>
+          <a href='./blog-post.html'><button class='tell-me-button'>JULY 28</button></a>
+          <h3 class='date-title'>AND ANOTHER <span id='blue-color'>POST</span></h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates culpa pariatur eos, tempora
+            nihil
+            maiores, ullam omnis impedit libero eveniet deleniti atque error?</p>
+        </div>
+        <div class="dates">
+          <div class='date-post'>
+          </div>
+          <a href='./blog-post.html'><button class='tell-me-button'>JULY 24</button></a>
+          <h3 class='date-title'>ONE MORE <span id='blue-color'>POST</span></h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates culpa pariatur eos, tempora
+            nihil
+            maiores, ullam omnis impedit libero eveniet deleniti atque error?</p>
+        </div>
+        <hr>
+        <div class='something-post'>
+
+          <div>
+            <h3 class='something-title'>SOMETHING OF NOTE</h3>

```

Comment:
Your images may be floating (I see in your CSS you applied a float to the image; good move).  But, like I said in the comment above about your `sidebar` as a whole, the space in which your `<p>` tag text lives is so narrow, there's no way to truly see that in action.  Resize your column, and you'll see what I mean.  It's set to float, but image is almost as wide as the column.